### PR TITLE
CEDS-1857 Shut MUCR summary page

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: app/controllers/consolidations/ShutMucrSummaryController.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      case Some(data) => Ok(shutMucrSummaryPage(data))
+      case None       => throw IncompleteApplication
+    }
+  }
+
+  def submit(): Action[AnyContent] = (authenticate andThen journeyType).async { implicit request =>
+    cacheService.fetchAndGetEntry[ShutMucr](movementCacheId(), ShutMucr.formId).flatMap {
+      case Some(shutMucr) =>
+        submissionService.submitShutMucrRequest(shutMucr, request.eori).flatMap { _ =>
+          cacheService.remove(movementCacheId()).map { _ =>
```
Comment (bambuchaAdm) [inline app/controllers/consolidations/ShutMucrSummaryController.scala]: Is removing from keystore is mandatory to make redirection ? I have i mind something like
```
submissionService.submitShutMucrRequest(shutMucr, request.eori).flatMap { _ =>
  Redirect(routes.ShutMucrConfirmationController.displayPage())
              .flashing(FlashKeys.MUCR -> shutMucr.mucr)
}.andThen {
  case Success(_) => cacheService.remove(movementCacheId())
}
```

File: app/models/requests/JourneyRequest.scala [hunk 2]
```diff
@@ -19,4 +19,7 @@ package models.requests
 import forms.Choice
 import play.api.mvc.WrappedRequest
 
-case class JourneyRequest[A](authenticatedRequest: AuthenticatedRequest[A], choice: Choice) extends WrappedRequest[A](authenticatedRequest)
+case class JourneyRequest[A](authenticatedRequest: AuthenticatedRequest[A], choice: Choice) extends WrappedRequest[A](authenticatedRequest) {
+
+  val eori = authenticatedRequest.user.eori
```
Comment (bambuchaAdm) [inline app/models/requests/JourneyRequest.scala]: :+1:

File: app/controllers/consolidations/ShutMucrSummaryController.scala (hunk 1 above)
Comment (PatrykRudnicki) [reply app/controllers/consolidations/ShutMucrSummaryController.scala]: I can change it, I took it from old implementation probably :)

Comment (PatrykRudnicki) [reply app/controllers/consolidations/ShutMucrSummaryController.scala]: Fixed

**Comment by bambuchaAdm** [comment]
As duplication of #51

**Pull request merged**